packages/itmat-interface/package.json

```diff
@@ -30,6 +30,7 @@
         "itmat-utils": "1.1.8",
         "mongodb": "3.5.5",
         "multer": "2.0.0-beta.1",
+        "nodemailer": "^6.4.6",
```

Comment:
Same comment as earlier, lets make sure we have pinned dependencies

Reply:
> I'm not certain how useful this notion of `forgetUsername` is ? Are we expecting people to know their password but not their username. 

Definitely should let people retrieve their username via email